package.json

```diff
@@ -13,7 +13,7 @@
     "request-promise": "4.2.6"
   },
   "devDependencies": {
-    "jasmine-reporters": "2.3.2",
-    "jasmine": "3.6.4"
+    "jasmine": "^3.7.0",
```

Comment:
should we be putting the '^' in front of the versions? If we're adding them here, I would think we'd want to add them to the other version numbers in this file as well

Reply:
It's pretty unusual to see all the dependencies use the same matching pattern, it usually varies from dependency to dependency. I think this is OK, actually.